zeeguu_api/app.py

```diff
@@ -25,7 +25,7 @@
 
 try:
     import dashboard
-    dashboard.config.from_file('/home/travis/build/zeeguu-ecosystem/Zeeguu-API/dashboard_test.cfg')
+    dashboard.config.from_file('/home/mircea/zee/http/api/dashboard.cfg')
```

Comment:
@kloostert -- this seems to still be hardcoded, and not using the envvar, right ? :)

Reply:
Right, I reverted that line back to what it was before. 
But specifying envvar `DASHBOARD_CONFIG` will override this location string.
Thus, whenever the envvar is specified, it will use that one, else the location string provided in the from_file() call.

Reply:
coool! but if we use the envvar then i think we should drop completely this hardcoded string... its's security paranoia... but, we are giving away info about the state of our server :)

Reply:
Also, a note should be added here about the envvar: https://github.com/flask-dashboard/Flask-Monitoring-Dashboard

Reply:
Ok, added some info about it in the setup section of the readme (on the dev branch).
The line which configures the dashboard can be set to `dashboard.config.from_file(None)`.
This will take the envvar instead of the location string.